src/storage/txn/commands/scan_stale_locks.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+use crate::storage::txn::commands::{Command, CommandExt, TypedCommand};
+use tikv_util::collections::HashMap;
+use txn_types::{Key, TimeStamp};
+
+command! {
+    /// Scan locks for resolving according to `txn_status`.
+    ///
+    /// During the GC operation, this should be called to find out stale locks whose timestamp is
+    /// before safe point.
+    /// This should follow by a `ResolveLock`
+    ScanStaleLocks:
```

Comment:
Tell me if you had a better name for this.

Reply:
Sounds good

Reply:
As this command will schedule a `ResolveLock` command, what about calling it `ScanAndResolveLocks`?

Reply:
ResolveLocksReadPhase?

Reply:
> As this command will schedule a `ResolveLock` command, what about calling it `ScanAndResolveLocks`?

I've also consider about this. But `ResolveLock` **is not part of this command**.

Reply:
> ResolveLocksReadPhase?

This one sounds good.